util/constants/quality-id-to-measure-map.js

```diff
@@ -0,0 +1,22 @@
+// Libraries
+var keyBy = require('lodash/keyBy');
+// Data
+var measures = require('../../measures/measures-data.json');
+
+/**
+ * @type {{}} - mapping of integer qualityIds to corresponding measure
+ */
+var QUALITY_ID_TO_MEASURE_MAP = keyBy(measures, function(measure) {
+  /**
+   * NOTE(sung): The qualityId is usually a string integer.
```

Comment:
I can probably fix this by just removing leading zeros instead of using `parseInt`.